news/20200706.feature

```diff
@@ -0,0 +1 @@
+add the top level cmakelists.txt file generation to the build command.
```

Comment:
Be sure to use capital letters when starting sentences. This file is used as-is to create a user-readable description of changes (main audience is end users, not developers).